README.md

```diff
@@ -1,57 +1,53 @@
 # calculator-angular-tdd
 
-Construyendo una aplicación usando la metodología tdd o test driven development, intentando utilizar las mejores prácticas conocidas por los autores [JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg) 
+Builiding a app using the metology tdd or test driven development, We have used the best practice that thea autors know ([JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg))
 
+We espect have all mecanisme related of continuos integration and continus delivey CI/CD
 
-
-Se espera tener todos los mecanismos de pruebas, integración y despliegue con CI/CD
-
-* La documentación que no requiera gráfico se realizará en .md dentro de este git
+* The domcuntation that not require grahps would made wit MarckDown inside for this git. 
 
 ## Sprint 1
 
-En el sprint 1 se prentendrá tener en producción la aplicación en alguna nube en producción.
+In sprint 1 we will pretend to have a production app inside a one cloud.
 
-### Acerca del producto
+### About the project
 
-Es un aplicación web con la funcionalidad de calculadora, dentro de las funciones de la calculadora se incluyen las operaciones de multiplicar, dividir, sumar, restar y tambien soportará operaciones con decimales.
-
-Se basará sobre una apliacación creada en otra tecnología [JuanSeBestia/react-calculator](https://github.com/JuanSeBestia/react-calculator), diponible en linea aquí FALTA_LINK
+This proyect is inspired in other proyect maded in other framework [JuanSeBestia/react-calculator](https://github.com/JuanSeBestia/react-calculator), see the onlie demo in [DEMO REACT CALCULATOR](https://juansebestia-react-calculator.firebaseapp.com/)
 
 ### Histoias de usuario
 
-Hisotrias y requerimientos de la apliacación [Requeriments_UserStories.md](Requeriments_UserStories.md)
+Histories and requirements of this app: [Requeriments_UserStories.md](wiki/Requeriments_UserStories.md)
 
 
 ## Sprint 2
 
-En el sprint 2 se presntará el mismo producto pero con una funcionalidad adicional de un back, ambos viviendo en la nube.
-
+In sprint 2 we add a one functionaly of the same producto, the new functionality would have had a backend service. The FrontEnd and Backend will coexist in the seame cloud.
 
-## Acerca de las tenologías
+## About of tecnologies
 
-Se emplearán tecnolgías en los que los autores oportunidades de aprender, se utilizará la estrategia de git Feature Branch
+We will use tecnologies wich autors have the oportunities to learn, We use the strategy of git called Git Feture Branch.
```

Comment:
technologies, which, authors would have the oportunities ... we will use the git strategy called Git Feature

Reply:
Done